src/pages/ServiceGraph/SummaryPanelNode.tsx

```diff
@@ -52,9 +53,9 @@ export default class SummaryPanelNode extends React.Component<SummaryPanelPropTy
     this.fetchRequestCountMetrics(this.props);
   }
 
-  componentWillReceiveProps(nextProps: SummaryPanelPropType) {
-    if (nextProps.data.summaryTarget && nextProps.data.summaryTarget !== this.props.data.summaryTarget) {
-      this.fetchRequestCountMetrics(nextProps);
+  componentDidUpdate(prevProps: SummaryPanelPropType) {
```

Comment:
Seems like this would not be necessary with Redux. If an interested property were changed it would re-render.

Reply:
I want to do a request to fetch new data to the server, but i don't want to do it always, only when is really needed.
If adding redux, this test would still apply i think, unless the redux part also takes care of the fetching the data (but this test would move to the redux container I think)